apps/src/turtle/turtle.js

```diff
@@ -202,6 +202,32 @@ Artist.prototype.injectStudioApp = function (studioApp) {
 };
 
 /**
+ * Initializes all sticker images as defined in this.skin.stickers, storing the
+ * created images in this.stickers.
+ *
+ * NOTE: initializes this.stickers as a side effect
+ *
+ * @return {Promise} that resolves once all images have finished loading,
+ *         whether they did so successfully or not.
+ */
+Artist.prototype.preloadAllStickerImages = function () {
+  this.stickers = {};
+
+  const loadSticker = name => new Promise(resolve => {
+    const img = new Image();
+
+    img.onload = () => resolve();
+    img.onerror = () => resolve();
```

Comment:
I think so; we definitely still want to resolve on error since we're gating the entire app on these. We could consider logging to newrelic